packages/@sanity/desk-tool/src/pane/EmptyListPane.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import DefaultPane from 'part:@sanity/components/panes/default'
+import styles from './styles/EmptyListPane.css'
+
+export default function EmptyListPane(props) {
+  return (
+    <DefaultPane title="Empty schema" isScrollable={false} staticContent="&nbsp;">
+      <div className={styles.root}>
+        <div className={styles.message}>
+          Your{' '}
+          <a
+            title="Schema documentation"
+            target="_blank"
+            rel="noopener noreferrer"
+            href="https://www.sanity.io/docs/content-studio/the-schema"
+          >
+            schema
+          </a>{' '}
+          does not contain any document types. If it did, those types would be listed here.
```

Comment:
This could perhaps be a bit more helpful/actionable? What about adding a link to how to get started with adding documents types?

Reply:
Oh, I see now that there's a link, but maybe make the text more actionable? _Read more about how to add your first schema type_, or smth?

Reply:
Good point! Fixed.